ts/defs.bzl

```diff
@@ -346,7 +355,7 @@ def ts_project(
         native.filegroup(
             name = typecheck_target_name,
             srcs = [tsc_target_name],
-            # This causes the types to be produced, which in turn triggers the tsc action to typecheck
+            # This causes the declarations to be produced, which in turn triggers the tsc action to typecheck
```

Comment:
This is reverting https://github.com/aspect-build/rules_ts/commit/9c3445e060f5db958b7285224e99d25354cc89b1#diff-fd5b55e792f846b8312680267efd413bf73bfe655630485e1514db77eb7d187cL363

Reply:
oh, I'll just change it back, no reason that was included in this PR